src/coreclr/tools/Common/Pgo/PgoFormat.cs

```diff
@@ -562,7 +562,9 @@ void MergeInSchemaElem(Dictionary<PgoSchemaElem, PgoSchemaElem> dataMerger, PgoS
                     switch (existingSchemaItem.InstrumentationKind)
                     {
                         case PgoInstrumentationKind.BasicBlockIntCount:
+                        case PgoInstrumentationKind.BasicBlockLongCount:
                         case PgoInstrumentationKind.EdgeIntCount:
+                        case PgoInstrumentationKind.EdgeLongCount:
```

Comment:
I missed this in #51625 so merging files with 64-bit counts currently does not work.

Reply:
Is there anything else you know of that would block us from using 64 bit counts across dynamic/static PGO?

I think we might want to update the class probe counter, but I don't think we use the value for anything right now, so perhaps it's not urgent. Changing this is a bit tricky as the runtime helper also knows the layout of this data.

Reply:
I'm not familiar with anything, no. At least I believe I have modified all places using `EdgeIntCount` and `BlockIntCount` in the code base now, but maybe something else could be hiding somewhere.

I do now see that I missed the class probe counter. I'll submit a PR for that.

Reply:
There is actually still this left: https://github.com/dotnet/runtime/blob/3b8adb707e40080b435003efcd9640e73ab2c039/src/coreclr/tools/aot/ILCompiler.ReadyToRun/JitInterface/CorInfoImpl.ReadyToRun.cs#L2456-L2457

I don't know if this is currently being used, however. Do we ever crossgen with instrumentation turned on?